src/filemenu_p.h

```diff
@@ -35,8 +35,10 @@ class AppInfoAction : public QAction {
         appInfo_{std::move(app)} {
         setToolTip(QString::fromUtf8(g_app_info_get_description(appInfo_.get())));
         GIcon* gicon = g_app_info_get_icon(appInfo_.get());
-        QIcon icon = Fm::IconInfo::fromGIcon(gicon)->qicon();
-        setIcon(icon);
+        const auto icnInfo = Fm::IconInfo::fromGIcon(gicon);
+        if(icnInfo) {
```

Comment:
I guess it should be ```if(icnInfo.get())```? It checks the content of the shared pointer rather than the pointer itself.